.idea/runConfigurations/mvnDebug.xml

```diff
@@ -1,15 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
We actually need this. Was implemented by Georgios to "Add the mvnDebug run configuration for IntelliJ"